controller.go

```diff
@@ -1,39 +1,71 @@
 package main
 
 import (
+	"io/ioutil"
 	"os"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
 type Controller struct {
 	k8sClient *kubernetes.Clientset
 
-	configStore *ConfigStore
-	epStore     *EpStore
+	bootstrapData *bootstrapData
+	configStore   *ConfigStore
+	epStore       *EpStore
 }
 
 func NewController(
 	k8sClient *kubernetes.Clientset,
 ) *Controller {
 	c := &Controller{
-		k8sClient: k8sClient,
+		k8sClient:   k8sClient,
+		configStore: NewConfigStore(k8sClient, os.Getenv(XDS_CONFIGMAP_ENV)),
 	}
 
-	c.configStore = NewConfigStore(k8sClient, os.Getenv(XDS_CONFIGMAP_ENV))
-	if err := c.configStore.Init(); err != nil {
-		panic(err)
-	}
+	bootstrapFile := os.Getenv(XDS_BOOTSTRAP_FILE)
```

Comment:
I'd also argue that I think we are going to want to expand this into some CLI flags. I know it's simple now, but we're probably going to need to pass along additional information like, path to envoy, and potentially flags to get passed along to envoy for the subprocess.